src/Fridge.java

```diff
@@ -1,57 +1,113 @@
 import components.map.Map;
 import components.map.Map1L;
 
-public class Fridge {
-    private boolean isOpen;
-    private static Map<String, Integer> fridgeItems;
+public class Fridge implements FridgeKernel, FridgeSecondary {
```

Comment:
Your hierarchy isn't quite right. It should be as follows:
1. Standard
2. FridgeKernel
3. Fridge
4. FrideSecondary
5. Fridge1